fe/fe-core/src/main/java/com/starrocks/catalog/UniqueConstraint.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+package com.starrocks.catalog;
+
+import com.clearspring.analytics.util.Lists;
+import com.google.common.base.Joiner;
+import org.spark_project.guava.base.Strings;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class UniqueConstraint {
```

Comment:
Maybe add some comments:
- DUP_KEYS's unique constraints are used as the hint, SR cannot guarantee the unique of real data.
- UNIQUE_KEY/PRIMARY_KEY's unique constraints are just the unique keys / primary keys (no need user add constraints?)